packages/app/src/overview/components/overview-screens/pure-overview/overview-content.component.tsx

```diff
@@ -29,7 +29,7 @@ interface OverviewContentProps {
   readonly checkForUpdateFailed: boolean
   readonly osVersion: string
   readonly caseColour: CaseColor | undefined
-  readonly lastBackupDate: Date
+  readonly lastBackupDate: Date | undefined
```

Comment:
I think this is subject for front-end meetings, why undefined not null? I don't know...